packages/component-library/stories/CardList.story.js

```diff
@@ -31,73 +17,13 @@ const CardRegistryMock = {
     Transit: 1,
     Demo: 1
   },
-  entries: [
-    {
+  entries: sampleFilterCardsWithTags.map(sampleCard => {
+    return {
       slug: "template-file-card",
```

Comment:
The slugs were different before, but now they're all `"template-file-card"`. This is just for a story, so it probably doesn't matter in this case, but if this were for production code, we'd want to make sure to dynamically generate this whole object that is returned so that the slug can be unique.